presentations/traits/slides.adoc

```diff
@@ -26,12 +26,12 @@ include::./1.rs[]
 `new` here is purely convention.
 
 == A Python analogy
-////
+--
```

Comment:
In case it's not intentional this speaker note block is missing the opening `[.notes]` marker.